### PR TITLE
Switch from Dep to Go modules

### PR DESCRIPTION
**Pull request by dennwc**
Switch from Dep to Go modules and build with Go 1.12.

Modules builds are repeatable, so the PR also drops `vendor` directory.

Also updates the SDK to the latest version.

Signed-off-by: Denys Smirnov <denys@sourced.tech>

### CODE REVIEW

**Comment by dennwc** [comment]
@creachadair I don't expect workflow to change. The user either pulls the Docker image and doesn't care about building from source, or uses `make build` which still works with modules. The only difference is that Go 1.11+ is now required.

**Comment by creachadair** [comment]
I'm not worried about it, but how could this change decrease coverage?

**Comment by dennwc** [comment]
I have no idea to be honest. The [details](https://codecov.io/gh/bblfsh/bblfshd/compare/031513f8bb5de5752d1bd1a2883a26cb16c7d5f7...ef394cc6073a0226e76fb2524fa993e95a866c3f) seems to suggest that only whitespace change to 2 lines occured. Their algorythm may use the total byte size to calculate the score, at least it's my guess.

**Comment by creachadair** [comment]
> I have no idea to be honest. The [details](https://codecov.io/gh/bblfsh/bblfshd/compare/031513f8bb5de5752d1bd1a2883a26cb16c7d5f7...ef394cc6073a0226e76fb2524fa993e95a866c3f) seems to suggest that only whitespace change to 2 lines occured. Their algorythm may use the total byte size to calculate the score, at least it's my guess.

That's certainly consistent with the size of the effect. Which I suppose only goes to reinforce my view that coverage is a sloppy metric and only marginally useful. 😀

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 language: go
 
 go:
-  - '1.10.x'
+  - '1.12.x'
```
Comment (bzz) [inline .travis.yml]: should not we be testing on 2 latest go version?

Comment (dennwc) [reply .travis.yml]: According to our [guide](https://github.com/src-d/guide/blob/master/engineering/conventions/go.md#supported-go-versions), applications support one latest stable Go version.

**Comment by dennwc** [comment]
Holding this off a bit. Will first cut a patch release with the updated drivers, and then the minor release with Go modules.

**Pull request merged**